python/tvm/hybrid/_internal.py

```diff
@@ -0,0 +1,62 @@
+"""Internal utilities for parsing Python subset to HalideIR"""
+
+import sys
+import inspect
+import numpy
+from ._intrin import HYBRID_GLOBALS
+from .. import api as _api
+from .. import make as _make
+from .. import expr as _expr
+from ..tensor import Tensor
+
+# Useful constants
+NOP = _make.Evaluate(_api.const(0, dtype='int32'))
```

Comment:
try not to construct IR object in global, since we sometimes will have runtime only object and we need to support non-dependency when we only import but did not use hybrid.

Construct a Constant class single and overload property to do it lazily when any of them is requested